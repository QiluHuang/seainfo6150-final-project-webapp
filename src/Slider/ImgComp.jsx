import React from 'react';
import styles from './ImgComp.module.css';

const ImgComp = ({ src }) => {
    let imgStyles = {
        width: 100 + "%",
        height: "auto"
    }
    return (
        <div className={styles.imgContainer}>
            <img src={src} alt="slide-img" className={styles.image}></img>
            {/* <img src={src} alt="slide-img" style={imgStyles}></img> */}
        </div>
    );
}

export default ImgComp;