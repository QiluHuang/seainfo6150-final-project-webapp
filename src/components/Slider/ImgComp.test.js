import React from "react";
import { render } from "@testing-library/react";
import ImgComp from "./ImgComp.jsx";
import img1 from '../../img/homepage_images/barbecue-pork-fried-rice.jpg';


describe("ImgComp tests", () => {
    it("renders the ImgComp component correctly", () => {
    const { container } = render(<ImgComp src={img1} />);
    expect(container).toMatchSnapshot();
    });
});