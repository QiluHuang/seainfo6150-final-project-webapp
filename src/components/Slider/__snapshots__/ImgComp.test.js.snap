// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImgComp tests renders the ImgComp component correctly 1`] = `
<div>
  <div
    class="imgContainer"
  >
    <img
      alt="slide-img"
      class="image"
      src="barbecue-pork-fried-rice.jpg"
    />
  </div>
</div>
`;
